.github/workflows/next-lint-check.yml

```
@@ -0,0 +1,31 @@
+name: Next.js Linting Check
+
+on:
+  pull_request:
+    branches:
+      - main
+      - master
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  lint:
+    name: Linting with Next.js
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v4
```
[v4](https://github.com/actions/checkout/releases/tag/v4.0.0) is latest

---

you Might wanna read this https://github.com/actions/checkout/issues/1487

tell me your opinons after, thanks!

---

I will run these locally and let you know the results!

---

> you Might wanna read this [actions/checkout#1487](https://github.com/actions/checkout/issues/1487)
> 
> tell me your opinons after, thanks!

@ashmit-coder I read it and it seems many people have referenced that issue.

> I will run these locally and let you know the results!

If it doesn't work, I suggest testing with [lower version](https://github.com/marketplace/actions/checkout?version=v3.6.0) (v3.6.0)